src/components/Board/index.js

```diff
@@ -133,7 +137,8 @@ function Board ({
             </Lane>
           ))}
         </DroppableBoard>
-        {allowAddLane && onNewLane && <LaneAdder onConfirm={addLane} />}
+        {renderLaneAdder ? renderLaneAdder({ addLane })
```

Comment:
I think `allowAddLane` must be considered even when the component receives a `renderLaneAdder`. What if the user want to render a custom lane and to control when it needs to be shown?